app/src/main/res/layout/project_view.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These xml files were created to resolve the references in the Adapter that were unresolved errors, changes will have to be made if implemented.

Reply:
Incomplete changes should not be merged to master.